README.md

```diff
@@ -20,6 +20,9 @@ The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.
     * `LOGDNA_HOSTNAME`: Alternative Host Name *(Optional)*  
     * `LOGDNA_TAGS`: Comma-separated Tags *(Optional)*  
     * `LOGDNA_URL`: Custom Ingestion URL *(Optional)*
+    * `RAW_EVENT_MESSAGE`: Setting `line` to Raw `event.message` *(Optional)*:
```

Comment:
Not a big fan of how this variable is named with the type it expects. Since it's looking for some boolean input I feel like the variable name should either reflect a question or an action instead of a noun. I think `LOG_RAW_EVENT` would be a better name that would make it easier to read.

Reply:
We should also think about the customer who is reading this. Do they know what `line` or `event.message` is? Is there a way we can explain this without referencing the code?

Reply:
@jakedipity, renamed it to `RAW_EVENTLOG` similar to the variables used for `logdna-s3`

The customers know what `event.message` is, which is the log shown on the `CloudWatch` dashboard. Whoever sets these things up, most probably, (or should) knows how `CloudWatch Logs` get passed into the `Lambda` functions.